core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
Yes, I will go back through all those.  In fact, I’m going to circle back and review everything recently done in last few days.  I’ve learned a couple of things here I didn’t know so I want to make sure I apply those correctly. 

 

I also reran some sonar checks and issues it use to complain about are not there now.  So obviously they were wrong at some point in time.  I’ve gotten so used to doing certain things that I take for granted what is right sometimes.

 

I’m getting ready to jump off for the night.  If possible, can you open a new issue with bullet points on what you want me to specifically look into (in case there are more)?  That might be useful in tracking it in case I get pulled off into life the next couple of days.  At the very least I don’t want to miss getting these back into their current state by Wednesday or so.

 

On the bright side, it was probably a wise decision to pull these in smaller chunks :)

 

Thanks,

 

Jeremy

 

From: Mark Lewis [mailto:notifications@github.com] 
Sent: Sunday, January 24, 2016 11:32 PM
To: psi-probe/psi-probe <psi-probe@noreply.github.com>
Cc: Jeremy Landis <jeremylandis@hotmail.com>
Subject: Re: [psi-probe] Add javabean tester and tests (#607)

 

In core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java <https://github.com/psi-probe/psi-probe/pull/607#discussion_r50653306> :

> @@ -34,6 +34,13 @@
>  
>    /**
>     * Instantiates a new url parser.
> +   */
> +  public UrlParser() {

I hate to ask, but do you mind going back through all the recent no-arg constructor additions and removals?

—
Reply to this email directly or view it on GitHub <https://github.com/psi-probe/psi-probe/pull/607/files#r50653306> .  <https://github.com/notifications/beacon/AA7ho4Zt795fKPnFGxJAamW8XCakIrVDks5pdZylgaJpZM4HLEob.gif>